test/e2e/marketplace_test.go

```diff
@@ -60,7 +60,7 @@ func TestMarketplace(t *testing.T) {
 	// run subtests
 	t.Run("marketplace-group", func(t *testing.T) {
 		t.Run("Cluster", MarketplaceCluster)
-		t.Run("csc-non-existing-namespace", runCSCWithNonExistingTargetNamespace)
+		t.Run("csc-tests", runCSCTests)
```

Comment:
Please don't collapse the tests. Each test should test a unique case and report failures on each.

Reply:
I am not opposed to grouping tests together, but the `runCSCTests` function should use the `t.Run(x,y)` method to call it's tests so the logs specify which test failed - the current approach will only print that the `csc-tests` failed and log the error.

Reply:
This is the point I was making earlier too but @kevinrizza seems to think that this the way to go. Thoughts?

Reply:
Yes we should use t.Run inside each group (subtests). The value there is that you can set up some resources if a set of tests are all going to test against them (i.e. if your tests are `CatalogSourceConfig` specific and they all need an `OperatorSource` to run, it is convenient to create that `OperatorSource` once upfront, run that subset of tests with `t.Run()` and then when `ctx.Cleanup()` is run for that test group the `OperatorSource` is deleted.